CONTRIBUTING.md

```diff
@@ -0,0 +1,162 @@
+# Contributing to the ForgeRock Web Login Framework
```

Comment:
Just adding a CONTRIBUTING.md to this project and starting with some questions that I've been asked and the answers I gave. This will at least get us started with some content regarding the development side of the framework.